### ISSUE TITLE
Keep path of tested files

### ISSUE THREAD

**Issue by ghost** [istanbuljs/puppeteer-to-istanbul#16]
It would be useful to keep the complete path of the tested files, especially when using es6 modules.

Now, in the Istanbul report, all files are shown as if they were in the root of the project. This is specially confusing when you have multiple files with same name in different directories (ie: "models/record.js", "views/record.js") - in this case all files but one get there name changed with an appended integer:

```
--------------|----------|----------|----------|----------|-------------------|
File          |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
--------------|----------|----------|----------|----------|-------------------|
All files     |       90 |      100 |      100 |       90 |                   |
 index.js     |      100 |      100 |      100 |      100 |                   |
 record-1.js  |    93.33 |      100 |      100 |    93.33 |                14 |
 record.js    |      100 |      100 |      100 |      100 |                   |
--------------|----------|----------|----------|----------|-------------------|
```

**Comment by vkoves** [comment]
Your request definitely makes sense. I've marked this as a bug and indicated that we would love contribution on resolving this issue

**Comment by ghost** [comment]
Cool. Let me have a go at it.

**Comment by ghost** [comment]
Fixed by #17.
